src/vsr/replica.zig

```diff
@@ -5623,12 +5619,10 @@ pub fn ReplicaType(
         fn is_repair(self: *const Self, message: *const Message.Prepare) bool {
             assert(message.header.command == .prepare);
 
+            if (message.header.view < self.view) return true;
+
```

Comment:
Would it make sense to further compress this down to a single expression? View is less or view is equal, satisfy is normal, and op is less?

Reply:
This=this entire function

Reply:
Ah yes, no other usages of this function anyway! Let me compress it down to a single expression and and use it directly in _on_prepare_.

Reply:
Oh, removing the function is even better!

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2143/commits/b4bac8022d9cfdac3bc8807459da8d932e6d3529